_data/changelogs/component-input-prefix-suffix.yml

```diff
@@ -2,6 +2,19 @@ title: Input prefix or suffix
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Moved `.usa-input-group--[width]` classes out of the `usa-form` package.
+    summaryAdditional: These classes are now generated in the `usa-input-prefix-suffix` package and can be used without the `.usa-form` parent element.
+    affectsStyles: true
+    githubPr: 6232
+    githubRepo: uswds
+    versionUswds: 3.11.0
+  - date: NNNN-NN-NN
```

Comment:
**chore**: chore: Add tomorrow's date.

```suggestion
  - date: 2024-12-18
```

Reply:
Updated in [a386105](https://github.com/uswds/uswds-site/pull/2991/commits/a386105edfb98a0e8f218fd6b8e4eada5dfb1e55)